sorting/buketSort.php

```diff
@@ -0,0 +1,42 @@
+function BucketSort(&$data)
```

Comment:
Per the [PSR-12 php style guide](https://www.php-fig.org/psr/psr-12/), every PHP file should have an opening `<?php` tag